File: android/app/src/main/java/net/ipv64/kivop/dtos/RideServiceDTOs/GetRiderDTO.kt
package net.ipv64.kivop.dtos.RideServiceDTOs

import java.util.UUID
import java.time.LocalDateTime

data class GetRiderDTO (
    var id : UUID,
    var userID : UUID,
    var username : String,
    var latitude : Float,
    var longitude : Float,
    var itsMe : Boolean,
    var accepted : Boolean,
)

File: DTOs/Sources/RideServiceDTOs/GetRiderDTO.swift
import Foundation

public struct GetRiderDTO: Codable {
    public var id: UUID
    public var userID: UUID
    public var username: String
    public var latitude: Float
    public var longitude: Float
    public var itsMe: Bool
    public var accepted: Bool
    
    public init(id: UUID, userID: UUID, username: String, latitude: Float, longitude: Float, itsMe: Bool, accepted: Bool) {
        self.id = id
        self.userID = userID
        self.username = username
        self.latitude = latitude
        self.longitude = longitude
        self.itsMe = itsMe
        self.accepted = accepted
    }
}
